website/snippets/_new-sl-setup.md

```diff
@@ -63,20 +72,35 @@ Note that:
-4. In the **Map new service token** section, map a service token to the credential you configured in the previous step. dbt Cloud automatically selects the service token permission set you need (Semantic Layer Only and Metadata Only).
-   - To add another service token, click **Add service token** under the **Linked service tokens** section.
 
-5. Click **Save** to link the service token to the credential. Remember to copy and save the service token securely, as it won't be viewable again after generation.
-<Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl-credential-created.jpg" width="100%" title="Manage multiple credentials and link them to service tokens for more granular control."/>
+3. In the **2. Map new service token** section, map a service token to the credential you configured in the previous step. dbt Cloud automatically selects the service token permission set you need (Semantic Layer Only and Metadata Only).
+
+4. To add another service token during configuration, click **Add Service Token**. 
+5. You can link more service tokens to the same credential later on in the **Semantic Layer Configuration Details** page. To add another service token to an existing Semantic Layer configuration, click **Add service token** under the **Linked service tokens** section.
+6. Click **Save** to link the service token to the credential. Remember to copy and save the service token securely, as it won't be viewable again after generation.
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl-credentials-service-token.jpg" width="90%" title="Use the configuration page to manage multiple credentials or link or unlink service tokens for more granular control."/>
+
+7. To delete a credential, go back to the **Credentials & service tokens** page.
+8. Under **Linked Service Tokens**, click **Edit**. Select **Delete Credential** to remove a credential.
 
-6. To delete a credential, go back to the **Semantic Layer & Credential**s page. Select **Delete credential** to remove a credential and click **Save**. 
-   
    When you delete a credential, any service tokens mapped to that credential in the project will no longer work and will break for any end users.
 
+## Delete configuration
+You can delete the entire Semantic Layer configuration for a project. Note that deleting the Semantic Layer configuration will remove all credentials and unlink all service tokens to the project. It will also cause all queries to the Semantic Layer to fail.
+
+Follow these steps to delete the Semantic Layer configuration for a project:
+
+1. Navigate to the **Project details** page.
+2. In the **Semantic Layer** section, select **Delete Semantic Layer**. 
+3. Confirm the deletion by clicking **Yes, delete semantic layer** in the confirmation pop up.
+
+To re-enable the dbt Semantic Layer in the future, you will need to recreate your configurations by following the [previous steps](#set-up-dbt-semantic-layer).
```

Comment:
Hiya @mirnawong1 

This is also another non-blocker suggestion, but would it be worth adding a sentence advising users if re-enabling the SL will have their previous work / projects saved or pick up where they left off?

Kind Regards
Natalie